core/src/test/scala/org/apache/spark/StatusTrackerSuite.scala

```diff
@@ -33,7 +33,7 @@ class StatusTrackerSuite extends SparkFunSuite with Matchers with LocalSparkCont
     val jobId: Int = eventually(timeout(10.seconds)) {
       val jobIds = jobFuture.jobIds
       jobIds.size should be(1)
-      jobIds.head
+      jobIds.sorted.reverse.head
```

Comment:
If the goal is to get the largest JobId, can we use `jobIds.max`?

Reply:
Good point.  Updated the code in this PR.